Advanced_Projects/PDFTextGenie/tests/test_pdftextgenie.py

```
@@ -0,0 +1,52 @@
+    class DummyEngine:
+        def __init__(self, *args, **kwargs):
+            pass
+        def recognize(self, images):
+            return 'dummy text'
+    monkeypatch.setattr('ocr.llama_ocr.EasyOCREngine', DummyEngine)
+    img = make_blank_image()
+    result = ocr_images([img], model='easyocr')
+    assert 'dummy text' in result
+
+def test_pdf_to_images_invalid():
+    with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)